### DISCUSSION TITLE
07.10.2024

### DISCUSSION THREAD

**Discussion by farag2** [farag2/Sophia-Script-for-Windows#590] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/farag2/Sophia-Script-for-Windows/releases/tag/6.7.0'>07.10.2024</a>.</em>

**Comment by YisusChrist** [comment]
How is Sophia related to WinterOS? I'm curious because I find it surprising that such a serious vulnerability and high security risk is reported here

**Reply by farag2** [reply]
> How is Sophia related to WinterOS?

We aren't related in any kind with this co-called project. I just decided to warn all users who are chasing for FPS boost with thier low-end PCs that there's a custom ISO with many reviews on YouTube which is being distributed with built-in backdoor with the only purpose to make a botnet within it,
